.github/workflows/linux.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       id: openvino_download
       with:
         platform: 'ubuntu22'
+        commit_packages_to_provide: 'wheels'
```

Comment:
I though that the right argument is "ov_artifact_name: 'openvino_wheels'", why whees? How user should know the right name of other artifacts?

Reply:
It's the name of the folder stored on shared drive, it's seen in OV workflow, plus you've added this folder by yourself :)